lib/db.c

```diff
@@ -9838,8 +9838,11 @@ grn_obj_get_nhooks(grn_ctx *ctx, grn_obj *obj, grn_hook_entry entry)
 }
 
 grn_obj *
-grn_obj_get_hook(
-  grn_ctx *ctx, grn_obj *obj, grn_hook_entry entry, int offset, grn_obj *hldbuf)
+grn_obj_get_hook(grn_ctx *ctx,
+                 grn_obj *obj,
+                 grn_hook_entry entry,
+                 int offset,
+                 grn_obj *data_holder_buf)
```

Comment:
How about using `data_buf` here?
```suggestion
                 grn_obj *data_buf)
```